### PR TITLE
Fixes

### PR DESCRIPTION
**Pull request by jazbar07**
Fixed the layout, again. Please, please, _please_... I beg of you... stop breaking it. 

After: (im like serious, if u break it again I will find you)

image

image

### CODE REVIEW

**Comment by rtogn** [comment]
Update: intentionally broke index layout in current PR

File: templates/index.html [hunk 1]
```diff
@@ -21,14 +21,13 @@
 
 	<main class="main_bar">
 		<div class="top_header">
-				<form action="/search_results" method="GET">
+				<form action="/search_results" method="GET" class="wrap">
 					<!-- search bar-->
 					<input type="text" name="term" class="search_text" placeholder="Search Youtube for Videos..">
-					<input class="search" type="submit" value="search" />
+					<button type="submit" class="search">Submit</button>
```
Comment (hafsa-hassan) [inline templates/index.html:27]: why did you change them to button? is that better?

**Review by hafsa-hassan — APPROVED** [review]

**Comment by hafsa-hassan** [comment]
I approved the merge
